styles.css

```diff
@@ -1,5 +1,15 @@
-/* font-family: 'Poppins', sans-serif; */
 /* UNIVERSAL----------------------------------------- */
+
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.hidden {
+	display: none;
+}
+
 /* HEADER---------------------------------------------*/
 
 header {
```

Comment:
my only comment on the styling is that it's generally anti-pattern to style by different tags, like element and class in the same file. The argument is that as css files get huge they become really challenging to manage without a dedicated system in place. Though you'll learn more on this later and I think at this stage just getting the CSS to do what you want it to do is a huge win. And this project looks great!